2020-01-24 10:27:43,024 [6] ERROR [en JBLV.Log.Log.RegistrarLog(NivelLog nivel, Exception ex)] 
System.NullReferenceException: Referencia a objeto no establecida como instancia de un objeto.
   en Mantenimiento.BusinessLayer.OrdenMasivaLogic.<InsertCorrectivo>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.BusinessLayer\OrdenMasivaLogic.cs:línea 75
2020-01-24 10:59:09,763 [39] ERROR [en JBLV.Log.Log.RegistrarLog(NivelLog nivel, Exception ex)] 
System.Exception: Desbordamiento de SqlDateTime. Debe estar entre 1/1/1753 12:00:00 AM y 12/31/9999 11:59:59 PM.
   en Mantenimiento.DataAccess.SolicitudRevisionTecnicaData.<InsertSolicitudRevisionTecnica_C>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.DataAccess\SolicitudRevisionTecnicaData.cs:línea 85
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   en Mantenimiento.BusinessLayer.OrdenMasivaLogic.<InsertCorrectivo>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.BusinessLayer\OrdenMasivaLogic.cs:línea 95
2020-01-24 11:01:28,293 [49] ERROR [en JBLV.Log.Log.RegistrarLog(NivelLog nivel, Exception ex)] 
System.Exception: Error en la solicitud procedimiento 'tb_SolicitudRevisionTecnica_C'. 'tb_SolicitudRevisionTecnica_C' es un objeto tabla.
   en Mantenimiento.DataAccess.SolicitudRevisionTecnicaData.<InsertSolicitudRevisionTecnica_C>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.DataAccess\SolicitudRevisionTecnicaData.cs:línea 85
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   en Mantenimiento.BusinessLayer.OrdenMasivaLogic.<InsertCorrectivo>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.BusinessLayer\OrdenMasivaLogic.cs:línea 95
2020-01-24 11:05:30,765 [35] ERROR [en JBLV.Log.Log.RegistrarLog(NivelLog nivel, Exception ex)] 
System.Exception: Error al convertir el valor del parámetro de String a DateTime.
   en Mantenimiento.DataAccess.ProgramacionData.<UpdateProgramacionOrdenCorrectivoGeneracion>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.DataAccess\ProgramacionData.cs:línea 119
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   en Mantenimiento.BusinessLayer.OrdenMasivaLogic.<InsertCorrectivo>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.BusinessLayer\OrdenMasivaLogic.cs:línea 130
2020-01-24 12:23:28,226 [86] ERROR [en JBLV.Log.Log.RegistrarLog(NivelLog nivel, Exception ex)] 
System.Exception: Error al convertir el valor del parámetro de String a DateTime.
   en Mantenimiento.DataAccess.ProgramacionData.<UpdateProgramacionOrdenCorrectivoGeneracion>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.DataAccess\ProgramacionData.cs:línea 119
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   en Mantenimiento.BusinessLayer.OrdenMasivaLogic.<InsertCorrectivo>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.BusinessLayer\OrdenMasivaLogic.cs:línea 130
2020-01-24 12:30:28,697 [68] ERROR [en JBLV.Log.Log.RegistrarLog(NivelLog nivel, Exception ex)] 
System.Exception: El procedimiento o la función 'usp_UDP_PROGRAMACION_OrdenCorrectivo_Generacion' esperaba el parámetro '@pCODI_PROGRAMACION', que no se ha especificado.
   en Mantenimiento.DataAccess.ProgramacionData.<UpdateProgramacionOrdenCorrectivoGeneracion>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.DataAccess\ProgramacionData.cs:línea 119
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   en Mantenimiento.BusinessLayer.OrdenMasivaLogic.<InsertCorrectivo>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.BusinessLayer\OrdenMasivaLogic.cs:línea 130
2020-01-24 12:31:52,351 [68] ERROR [en JBLV.Log.Log.RegistrarLog(NivelLog nivel, Exception ex)] 
System.Exception: El procedimiento o la función 'usp_UDP_PROGRAMACION_OrdenCorrectivo_Generacion' esperaba el parámetro '@pNumeroMantenimiento', que no se ha especificado.
   en Mantenimiento.DataAccess.ProgramacionData.<UpdateProgramacionOrdenCorrectivoGeneracion>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.DataAccess\ProgramacionData.cs:línea 119
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   en Mantenimiento.BusinessLayer.OrdenMasivaLogic.<InsertCorrectivo>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.BusinessLayer\OrdenMasivaLogic.cs:línea 130
2020-01-24 16:16:17,315 [18] ERROR [en JBLV.Log.Log.RegistrarLog(NivelLog nivel, Exception ex)] 
System.InvalidOperationException: TransactionScope se debe eliminar en el mismo subproceso en el que se creó.
   en System.Transactions.TransactionScope.Dispose()
   en Mantenimiento.BusinessLayer.OrdenMasivaLogic.<InsertCorrectivo>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.BusinessLayer\OrdenMasivaLogic.cs:línea 137
2020-01-24 16:16:55,941 [71] ERROR [en JBLV.Log.Log.RegistrarLog(NivelLog nivel, Exception ex)] 
System.Data.SqlClient.SqlException (0x80131904): Se agotó el tiempo de espera de ejecución. El período de tiempo de espera transcurrió antes de la finalización de la operación o el servidor no responde. ---> System.ComponentModel.Win32Exception (0x80004005): Tiempo de espera de la operación de espera agotado
   en System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   en System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   en System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   en System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   en System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   en System.Data.SqlClient.SqlDataReader.get_MetaData()
   en System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   en System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteReader()
   en Mantenimiento.DataAccess.SolicitudRevisionTecnicaData.ObtenerUltimoId() en D:\JELAF\MANTENIMIENTO\Template.DataAccess\SolicitudRevisionTecnicaData.cs:línea 28
   en Mantenimiento.BusinessLayer.OrdenMasivaLogic.<InsertCorrectivo>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.BusinessLayer\OrdenMasivaLogic.cs:línea 80
ClientConnectionId:62e5fffd-dc80-4025-9c48-049e921d1d96
Error Number:-2,State:0,Class:11
2020-01-24 16:18:39,926 [6] ERROR [en JBLV.Log.Log.RegistrarLog(NivelLog nivel, Exception ex)] 
System.InvalidOperationException: TransactionScope se debe eliminar en el mismo subproceso en el que se creó.
   en System.Transactions.TransactionScope.Dispose()
   en Mantenimiento.BusinessLayer.OrdenMasivaLogic.<InsertCorrectivo>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.BusinessLayer\OrdenMasivaLogic.cs:línea 137
2020-01-24 16:21:40,847 [71] ERROR [en JBLV.Log.Log.RegistrarLog(NivelLog nivel, Exception ex)] 
System.InvalidOperationException: TransactionScope se debe eliminar en el mismo subproceso en el que se creó.
   en System.Transactions.TransactionScope.Dispose()
   en Mantenimiento.BusinessLayer.OrdenMasivaLogic.<InsertCorrectivo>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.BusinessLayer\OrdenMasivaLogic.cs:línea 137
2020-01-24 16:22:33,659 [50] ERROR [en JBLV.Log.Log.RegistrarLog(NivelLog nivel, Exception ex)] 
System.InvalidOperationException: TransactionScope se debe eliminar en el mismo subproceso en el que se creó.
   en System.Transactions.TransactionScope.Dispose()
   en Mantenimiento.BusinessLayer.OrdenMasivaLogic.<InsertCorrectivo>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.BusinessLayer\OrdenMasivaLogic.cs:línea 137
2020-01-24 17:17:11,368 [86] ERROR [en JBLV.Log.Log.RegistrarLog(NivelLog nivel, Exception ex)] 
System.Exception: El procedimiento o la función 'usp_DEL_tb_Informe_Anular' esperaba el parámetro '@pNumeroInforme', que no se ha especificado.
   en Mantenimiento.DataAccess.InformeData.<AnularInforme>d__3.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.DataAccess\InformeData.cs:línea 152
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   en Mantenimiento.BusinessLayer.OrdenMasivaLogic.<AnularCorrectivo>d__2.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.BusinessLayer\OrdenMasivaLogic.cs:línea 176
2020-01-24 17:22:11,163 [48] ERROR [en JBLV.Log.Log.RegistrarLog(NivelLog nivel, Exception ex)] 
System.Exception: Desbordamiento de SqlDateTime. Debe estar entre 1/1/1753 12:00:00 AM y 12/31/9999 11:59:59 PM.
   en Mantenimiento.DataAccess.ProgramacionData.<UpdateProgramacionOrdenCorrectivoGeneracion>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.DataAccess\ProgramacionData.cs:línea 119
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   en Mantenimiento.BusinessLayer.OrdenMasivaLogic.<InsertPreventivo>d__3.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.BusinessLayer\OrdenMasivaLogic.cs:línea 291
2020-01-24 17:22:21,480 [86] ERROR [en JBLV.Log.Log.RegistrarLog(NivelLog nivel, Exception ex)] 
System.Exception: Desbordamiento de SqlDateTime. Debe estar entre 1/1/1753 12:00:00 AM y 12/31/9999 11:59:59 PM.
   en Mantenimiento.DataAccess.ProgramacionData.<UpdateProgramacionOrdenCorrectivoGeneracion>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.DataAccess\ProgramacionData.cs:línea 119
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   en Mantenimiento.BusinessLayer.OrdenMasivaLogic.<InsertPreventivo>d__3.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.BusinessLayer\OrdenMasivaLogic.cs:línea 291
2020-01-24 17:22:33,909 [48] ERROR [en JBLV.Log.Log.RegistrarLog(NivelLog nivel, Exception ex)] 
System.Exception: Desbordamiento de SqlDateTime. Debe estar entre 1/1/1753 12:00:00 AM y 12/31/9999 11:59:59 PM.
   en Mantenimiento.DataAccess.ProgramacionData.<UpdateProgramacionOrdenCorrectivoGeneracion>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.DataAccess\ProgramacionData.cs:línea 119
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   en Mantenimiento.BusinessLayer.OrdenMasivaLogic.<InsertPreventivo>d__3.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.BusinessLayer\OrdenMasivaLogic.cs:línea 291
2020-01-24 17:27:02,344 [70] ERROR [en JBLV.Log.Log.RegistrarLog(NivelLog nivel, Exception ex)] 
System.Exception: Desbordamiento de SqlDateTime. Debe estar entre 1/1/1753 12:00:00 AM y 12/31/9999 11:59:59 PM.
   en Mantenimiento.DataAccess.ProgramacionData.<UpdateProgramacionOrdenCorrectivoGeneracion>d__1.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.DataAccess\ProgramacionData.cs:línea 119
--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---
   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   en Mantenimiento.BusinessLayer.OrdenMasivaLogic.<InsertPreventivo>d__3.MoveNext() en D:\JELAF\MANTENIMIENTO\Template.BusinessLayer\OrdenMasivaLogic.cs:línea 291
